### ISSUE TITLE
Rename repo to eCLM_static-file-generator

### ISSUE THREAD

**Issue by mvhulten** [HPSCTerrSys/eCLM_static-file-generator#9]
This repo needs to be renamed as discussed in meetings before.

Can someone (@jjokella @kvrigor @DCaviedesV) give me this right?

With sufficient rights, this should turn up [here](https://github.com/HPSCTerrSys/eCLM_static_file_workflow/settings).

**Comment by DCaviedesV** [comment]
Right, sorry @mvhulten you mentioned this and I needed to check the privileges. This can only be done with admin privileges, which we should keep in the smallest group possible. I can rename the repo, but I'm afraid I cannot find the email where you mentioned the target name. Could you remind me?

**Comment by DCaviedesV** [comment]
@mvhulten are you happy with that name?

**Comment by mvhulten** [comment]
No, it should be `eCLM_static-file-generator`.  This is more consistent with other repos.

**Comment by s-poll** [comment]
I have renamed it now to `eCLM_static-file-generator`
